develop/components/common/TitleBar.tsx

```diff
@@ -109,18 +110,33 @@ export default class TitleBar extends React.Component<TitleBarProps, TitleBarSta
         this.setState({ searchInFocus: false, searchList: [] });
     }
 
+    /**
+     * Checks whether the input key is an `Escape` key
+     * @param {React.KeyboardEvent<HTMLInputElement>} event The keyboard event
+     * @returns {boolean} True if it is `Escape` key
+     */
+    isEscapeKey(event: React.KeyboardEvent<HTMLInputElement>): boolean {
+        if (event.key === "Escape" || event.key === "Esc") {
+            return true;
+        } else if (event.keyCode === 27) {
```

Comment:
Deprecated, avoid where possible

Reply:
This is old code, I just moved it around. I know keyCode is deprecated and I get the warning whenever I run the build. But this solution is put here because of some inconsistency I noticed with some browsers especially IE. So it's like a fallback if `event.key` didn't work